administrator/components/com_joomgallery/models/maintenance.php

```diff
@@ -1033,11 +1075,42 @@ public function setUser()
       return false;
     }
 
+    $edit_ids = array();
```

Comment:
I am not able to properly test this code here. I gave several pictures an unknown owner directly in the database and then let the maintenance manager run. It shows me the faulty images in the images tab but how is it possible to assign a valid user to the images? Please provide some test instructions.

Reply:
Thanks for your help but by using the batch job it is just possible to set the new user to 0, which means 'No user'.
I had a closer look to the problem and found a bug, see #94 .